pmm/pmm2-server-release.groovy

```diff
@@ -19,6 +19,10 @@ pipeline {
             defaultValue: 'perconalab/pmm-server:dev-latest',
             description: 'pmm-server container version (image-name:version-tag)',
             name: 'DOCKER_VERSION')
+        string(
+            defaultValue: '',
+            description: 'OVA image filename',
```

Comment:
Image Tag?

Reply:
@puneet0191 I can not say that it is ova image tag, we specify image name but not a tag.